apps/tools/src/app/main.ts

```diff
@@ -1,13 +1,13 @@
-import {RemoveExtensions} from './removeExtensions';
-import {AddPermission, RemovePermission} from './permissions';
+import { RemoveExtensions } from './removeExtensions';
+import { AddPermission, RemovePermission } from './permissions';
 import * as Yargs from 'yargs';
-import {RemoveExtras} from './removeExtras';
-import {UserListCommand} from './user-list';
-import {PopulateFromAuth0} from './populateFromAuth0';
-import {Migrate} from './migrate';
-import {ChangeId, ChangeIdOptions} from './change-id';
-import { positionElements } from '@ng-bootstrap/ng-bootstrap/util/positioning';
+import { RemoveExtras } from './removeExtras';
```

Comment:
It's not a big deal... but we should figure out what the options is in WebStorm that would allow consistency with spacing between curly braces on imports. `import { RemoveExtras };` vs. `import {RemoveExtras};`